src/pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -466,6 +490,10 @@ def __init__(self, extra_options):
         # Note: this check is currently performed on full cells, but is loosened for
         # half-cells where you must pass a tuple of options to only set MSMR models in
         # the working electrode
+        _ensure_tuple(options["open-circuit potential"])
+        _ensure_tuple(options["particle"])
+        _ensure_tuple(options["intercalation kinetics"])
```

Comment:
```suggestion
        options["open-circuit potential"] = _ensure_tuple(options["open-circuit potential"])
       options["particle"] = _ensure_tuple(options["particle"])
        ptions["intercalation kinetics"] = _ensure_tuple(options["intercalation kinetics"])
```

Reply:
If true we must then rename the helper function to something like `_make_tuple` instead

Reply:
Yep! This is not editing things in-place.